.github/actions/install-dashboards/action.yml

```diff
@@ -42,7 +40,6 @@ runs:
     - uses: actions/checkout@v2
```

Comment:
I am not sure if you need the `mkdir -p plugins` step here. I thought that dashboards would already have a plugin folder when it is build like opensearch core. 

You would know better than I though

Reply:
Ya the Dashboards itself has the `plugin` directory: https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/plugins